docs/BR.md

```diff
@@ -1493,124 +1498,124 @@ Certificates MUST be of type X.509 v3.
+d. `basicConstraints`
 
-f. nameConstraints (optional)
+   This extension MUST be present and MUST be marked critical. The cA field MUST be set true. The pathLenConstraint field MAY be present.
 
-> If present, this extension SHOULD be marked critical\*.
->
-> \* Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
+e. `keyUsage`
 
-g. extkeyUsage (optional)
+   This extension MUST be present and MUST be marked critical. Bit positions for `keyCertSign` and `cRLSign` MUST be set. If the Subordinate CA Private Key is used for signing OCSP responses, then the `digitalSignature` bit MUST be set.
 
-> For Subordinate CA Certificates to be Technically constrained in line with section 7.1.5, then either the value id-kp-serverAuth [RFC5280] or id-kp-clientAuth [RFC5280] or both values MUST be present\*\*.
->
-> Other values MAY be present.
->
-> If present, this extension SHOULD be marked non-critical.
->
-> \*\* Generally Extended Key Usage will only appear within end entity certificates (as highlighted in RFC 5280 (4.2.1.12)), however, Subordinate CAs MAY include the extension to further protect relying parties until the use of the extension is consistent between Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
+f. `nameConstraints` (optional)
 
+   If present, this extension SHOULD be marked critical[^*].
+
+[^*]: Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
+
+g. `extkeyUsage` (optional)
+
+   For Subordinate CA Certificates to be Technically constrained in line with section 7.1.5, then either the value `id-kp-serverAuth` [RFC5280] or `id-kp-clientAuth` [RFC5280] or both values MUST be present[^**].
+
+   Other values MAY be present.
+
+   If present, this extension SHOULD be marked non-critical.
+
+[^**]: Generally Extended Key Usage will only appear within end entity certificates (as highlighted in RFC 5280 (4.2.1.12)), however, Subordinate CAs MAY include the extension to further protect relying parties until the use of the extension is consistent between Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
 
 #### 7.1.2.3 Subscriber Certificate
-a. certificatePolicies
+a. `certificatePolicies`
+
+   This extension MUST be present and SHOULD NOT be marked critical.
+
+   *  `certificatePolicies:policyIdentifier` (Required)
```

Comment:
You used two spaces after the bullet here, but only one space after the bullet in 7.1.2.2

This threw me off as I was trying to wonder if my eyes were going wonky on me re: the indentation 😅

Reply:
Heh! Oops. Using one works fine, so I fixed it everywhere to use one space for consistency (bb0feb9).